app/src/main/res/values/strings.xml

```diff
@@ -685,5 +685,9 @@
     <string name="no_internet_title">Internet Connection Lost</string>
     <!-- This string is displayed in the message of the notification message shown when the device loses internet connection. -->
     <string name="no_internet_message">Check your device configuration to fix the problem.</string>
-
+    <!-- This string is displayed in the title of the alert dialog shown when the user does not have permission to run the application on the device. -->
+    <string name="not_entitled_title">Unable To Run</string>
+    <!-- This string is displaye in the message body of the alert dialog shown when the user does not have permission to run the application on the device.
+         '%1$s' will be replace at runtime with the app's name. -->
+    <string name="not_entitled_message">%1$s does not have permission to run on this device.</string>
```

Comment:
Update the explanation string for Unable to Run to say:
"%1$s does not have permission to run on this device and will now exit."